### ISSUE TITLE
resume should be available

### ISSUE THREAD

**Issue by mubasshir** [johnschult/jquery.countdown360#13]
Hi,
I feel ```resume()``` should be available to control. Is it there? because I did not find in API

**Comment by mubasshir** [comment]
no update here?

**Comment by js798p** [comment]
You are right, there is no `resume` function. Feel free to implement it though and submit a PR :smile:

**Comment by i-am-chris** [comment]
I am into it too, any solution to this?
